pickee_api/views.py

```diff
@@ -93,7 +93,18 @@ def user_signup(request):
 
 def user_logout(request):
     logout(request)
-    return redirect('index')
+    return redirect('home')
+
+def profile_main(request):
```

Comment:
I think we should just call this view `profile` to avoid confusion.

Reply:
Ok.
This naming was used because we have preferences and history that technically also relate to the profile.
But, no problem, I will change it to "profile, preferences, history"